content/actions/guides/building-and-testing-python.md

```diff
@@ -421,14 +427,23 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install setuptools wheel twine
-      - name: Build and publish
-        env:
-          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
-          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+          pip install build
+      - name: Build package
         run: |
-          python setup.py sdist bdist_wheel
-          twine upload dist/*
+          python -m build
+      - name: Publish DEV package
+        if: "github.event.release.prerelease"
+        uses: pypa/gh-action-pypi-publish@27b31702a0e7fc50959f5ad993c78deac1bdfc29
+        with:
+          user: __token__
+          password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+          repository_url: https://test.pypi.org/legacy/
```

Comment:
I'm not sure this is the right flow.  Pre-releases (e.g. v1.0.0-alpha1) should still go to production PyPI.  You would want to publish to test PyPI to smoke out issues with your release flow -- that's something you can do on every commit.

Reply:
So I would just skip this step and run the next step without the `if`.

Reply:
ผมทำไม่ค่อยเป็น

Reply:
Thanks @brcrista - I've changed this now under this commit: https://github.com/github/docs/pull/6090/commits/fdb24c003a6fde54049f326d12d943b253e9a12b